server/models/workspaceModel.js

```diff
@@ -26,6 +42,7 @@ const workspaceSchema = mongoose.Schema(
       unique: true,
     },
     boards: [boardSchema],
+    columns: [columnSchema],
```

Comment:
You don't have to change the workspace schema. Workspaces don't explicitly need to know about the columns in their boards, if a workspace wants to look at its columns it would go into a specific board to find it anyways. So I'd remove this line :)

Reply:
Fixed!